purchases_ui_flutter/android/src/main/kotlin/com/revenuecat/purchases_ui_flutter/PurchasesUiFlutterPlugin.kt

```diff
@@ -184,10 +205,87 @@ class PurchasesUiFlutterPlugin: FlutterPlugin, MethodCallHandler, ActivityAware,
             ?: PaywallSource.DefaultOffering
     }
 
+    private fun createCustomerCenterListener(): CustomerCenterListenerWrapper {
+        return object : CustomerCenterListenerWrapper() {
+            override fun onRestoreStartedWrapper() {
+                channel.invokeMethod("onRestoreStarted", null)
+            }
+
+            override fun onRestoreCompletedWrapper(customerInfo: Map<String, Any?>) {
+                channel.invokeMethod("onRestoreCompleted", customerInfo)
+            }
+
+            override fun onRestoreFailedWrapper(error: Map<String, Any?>) {
+                channel.invokeMethod("onRestoreFailed", error)
+            }
+
+            override fun onShowingManageSubscriptionsWrapper() {
+                channel.invokeMethod("onShowingManageSubscriptions", null)
+            }
+
+            override fun onFeedbackSurveyCompletedWrapper(feedbackSurveyOptionId: String) {
+                channel.invokeMethod(
+                    "onFeedbackSurveyCompleted",
+                    mapOf(
+                        "optionId" to feedbackSurveyOptionId
+                    )
+                )
+            }
+
+            override fun onManagementOptionSelectedWrapper(action: String, url: String?) {
+                channel.invokeMethod(
+                    "onManagementOptionSelected",
+                    mapOf(
+                        "optionId" to action,
+                        "url" to url
+                    )
+                )
+            }
+
+            override fun onManagementOptionSelectedWrapper(
+                action: String,
+                customAction: String?,
+                purchaseIdentifier: String?
+            ) {
+                // DEPRECATED: This method is deprecated and replaced by onCustomActionSelectedWrapper
+                // No-op implementation to maintain compatibility
+            }
+
+            override fun onCustomActionSelectedWrapper(
+                actionId: String,
+                purchaseIdentifier: String?
+            ) {
+                channel.invokeMethod(
+                    "onCustomActionSelected",
+                    mapOf(
+                        "actionId" to actionId,
+                        "purchaseIdentifier" to purchaseIdentifier
+                    )
+                )
+            }
+        }
+    }
+
+    private fun ensureCustomerCenterListener() {
```

Comment:
Hmm the name for this one is weird... maybe `setCustomerCenterListenerIfNeeded`?